.github/workflows/ci.yml

```diff
@@ -123,6 +123,11 @@ jobs:
         with:
           command: install
           args: --debug --path ./forc
+      - name: Install Forc plugins
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: --debug forc-fmt forc-lsp forc-explore
```

Comment:
would we want to document changes from the latest release or from the local crates that we get from `--path`, or is this a non-issue?

Reply:
I think we should do the same as we do for `forc`, i.e. from `--path`. We can worry about the behaviour of `latest` w.r.t. selecting the latest published release rather than `master` in #1577.